Hookies/Game/Engine/GameEngine.swift

```diff
@@ -267,12 +324,15 @@ class GameEngine {
         })
     }
 
-    private func startGame() {
+    private func startCountdown() {
```

Comment:
`startCountdown()` should be doing only 1 thing which is to call `delegate?.didStartCountdown()`. Methods like `connectToGame()`, `subscribeToOtherPlayersState ()`, `setCurrentPlayer()` shouldn't be calling `startCountdown()` because this will lead to confusion for other developers. If we can have an `update()` method and inside `update()` you can call `startCountdown()` then it would better. Because only within 1 place is calling the `startCountdown()` method.